### DISCUSSION TITLE
Support for other languages

### DISCUSSION THREAD

**Discussion by DIABL06666** [Domtaro/SR2Control#11] [Q&A] [answered]
Is it possible to add the Polish language?

**Comment by Domtaro** [comment, marked as answer]
> Is it possible to add the Polish language?

I guess it is possible but, It is not easy.
We need to check 2 points to support the other languages.

1. If SR (Speech Recognition) system supports that language
  For example, I recommending to use Yukarinette Connector NEO as standard configuration for SR . And it seems to support multiple languages (include Polish)

2. If it's possible that you build-up a Grammar for that language
  We need to build-up the mappings for words-orders-commands (I call it Grammar) to use SR2Control.
  That completely depends on users effort.
  I'm not sure if I can support you enough to build that.

So, you need to consider especially 2. if you want to use SR2Control with other languages.
